### ISSUE TITLE
Regression: `LiteBuildConfig.output_dir` no longer honored (now defaulting to `_output` regardless of setting)

### ISSUE THREAD

**Issue by posita** [jupyterlite/jupyterlite#849]
Config (same behavior whether this is in `juptyer_lite_config.json` or in `overrides.json`):

``` json
{
  "LiteBuildConfig": {
    "contents": ["docs/notebooks"],
    "federated_extensions": [
      "https://pypi.io/packages/py3/J/Jinja2/Jinja2-3.1.2-py3-none-any.whl",
      "https://pypi.io/packages/py3/j/jupyterlab-widgets/jupyterlab_widgets-3.0.3-py3-none-any.whl"
    ],
    "output_dir": "docs/jupyter",
    "ignore_sys_prefix": ["federated_extensions", "mathjax"],
    "piplite_urls": []
  }
}
```

Despite the above configuration (which worked fine in the past), `jupyter lite build` results in docs being built in `./_output`.

``` sh
% jupyter lite --version
0.1.0b13
% ls -d _output docs/jupyterlite
ls: cannot access '_output': No such file or directory
ls: cannot access 'docs/jupyterlite': No such file or directory
docs
% jupyter lite build
...
% \ls -d _output docs/jupyterlite
ls: cannot access 'docs/jupyterlite': No such file or directory
_output
docs
```

The docs are not (or at least no longer) clear on where/how this is supposed to be configured.

**Comment by bollwyvl** [comment]
This is a security-related fix inherited from jupyter_core. The jupyterlite
0.1.0b14 release has workaround. If you can't update, I believe there is a
--config flag, but ideally we'll have some behavior that didn't require
that moving forward.

**Comment by jtpio** [comment]
Yes this was noticed and fixed in https://github.com/jupyterlite/jupyterlite/pull/844, released in `0.1.0b14`.

@posita you can try to update to the latest release with `pip install --pre -U jupyterlite` before re-running `jupyter lite build`.

**Comment by posita** [comment]
Yup! I can verify this is working again with `b14`. Thanks!
